packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -1,5 +1,6 @@
 /* External Imports */
 import { Logger, injectL2Context } from '@eth-optimism/core-utils'
+import { createWriteStream } from 'pino-sentry'
```

Comment:
We should maybe make this used in `core-utils`'s `logger.ts`?

Reply:
^yeah this might be a good idea! pino-sentry uses an older version of pino that i can't consistently overwrite, so i wanted to test out the differences between bs (uses this older version with logging) and dtl (uses the express plugins without logging) during this initial phase, while figuring out the versioning headache

Reply:
seems like this still isn't resolved: https://github.com/yarnpkg/yarn/issues/4874

Reply:
since i can't update to latest sentry, thoughts on keeping dtl on regular sentry for now and testing out `pino-sentry` through batch submitter? i'm down to tweak this as we go, just don't know what features we might be missing out on yet. i can also make a PR or fork `pino-sentry` to update

Reply:
I think the right approach is to make a PR upstream and update their dep, if not a lot of work?

Reply:
sounds good will do!